### PR TITLE
--precisions flag for downloader.py

### PR DESCRIPTION
**Pull request by dkurt**
resolves https://github.com/opencv/open_model_zoo/issues/308

### CODE REVIEW

File: tools/downloader/README.md [hunk 1]
```diff
@@ -59,6 +59,13 @@ option:
 The `--all` option can be replaced with other filter options to download only
 a subset of models. See the "Shared options" section.
 
+You may use `--precisions` flag to specify comma separated precisions of weights
+to be downloaded.
+
+```sh
+./downloader.py --name face-detection-retail-0004 --precision FP16,INT8
```
Comment (grib0ed0v) [inline tools/downloader/README.md]: @dkurt `--precisions`, otherwise it's not working

Comment (dkurt) [reply tools/downloader/README.md]: @grib0ed0v, nice catch, thank you!

File: tools/downloader/downloader.py [hunk 2]
```diff
@@ -191,6 +191,8 @@ def main():
         help='download only models whose names match at least one of the patterns in the specified file')
     parser.add_argument('--all',  action='store_true', help='download all available models')
     parser.add_argument('--print_all', action='store_true', help='print all available models')
+    parser.add_argument('--precisions', metavar='PREC[,PREC...]',
```
Comment (eaidova) [inline tools/downloader/downloader.py]: why not `nargs`?  it allows setting list of arguments using space as delimiter. 
https://docs.python.org/3/library/argparse.html#nargs

Comment (dkurt) [reply tools/downloader/downloader.py]: To align it with `converter.py`:

https://github.com/opencv/open_model_zoo/blob/c3b4b2b9845db2ca15e03df9804b6f98d7a3c211/tools/downloader/converter.py#L98-L99

**Comment by kchechil** [comment]
Jenkins please retry a build

**Pull request merged**